client/src/components/Admin/Jobs/index.vue

```diff
@@ -94,9 +101,78 @@ export default {
             loading: false,
             jobList: [],
             root: process.env.VUE_APP_ROOT_API,
+            createJobModal: false,
```

Comment:
we need to use the component for update as well
so giving name `createJobModal` will be wrong
give appropriate naming suitable for both cases